src/test/librados/c_read_operations.cc

```diff
@@ -607,6 +614,48 @@ TEST_F(CReadOpsTest, Stat) {
   rados_release_read_op(op);
 }
 
+TEST_F(CReadOpsTest, Stat2) {
+  rados_read_op_t op = rados_create_read_op();
+  uint64_t size = 1;
+  int rval;
+  rados_read_op_stat2(op, &size, NULL, &rval);
+  EXPECT_EQ(-ENOENT, rados_read_op_operate(op, ioctx, obj, 0));
+  EXPECT_EQ(-EIO, rval);
+  EXPECT_EQ(1u, size);
+  rados_release_read_op(op);
+
+  struct timespec ts;
+  ts.tv_sec = 1457129052;
+  ts.tv_nsec = 123456789;
+  rados_write_op_t wop = rados_create_write_op();
+  rados_write_op_write(wop, data, len, 0);
+  EXPECT_EQ(0, rados_write_op_operate2(wop, ioctx, obj, &ts, 0));
```

Comment:
if the following steps rely on the completion of this call, we should use `ASSERT_EQ()` here, like
```c++ 
ASSERT_EQ(0, rados_write_op_operate2(wop, ioctx, obj, &ts, 0));
```

so that the test bails out if `rados_write_op_operate2()` fails.